packages/elements/src/icon/index.ts

```diff
@@ -93,15 +93,20 @@ export class Icon extends BasicElement {
 
   private _src: string | null = null;
   /**
-   * Src location of an svg icon.
+   * Src location of a svg icon.
```

Comment:
svg starts with a vowel sound so an svg is already correct.
```suggestion
   * Src location of an svg icon.
```